Trading/Mode/dca_trading_mode/dca_trading.py

```diff
@@ -90,12 +90,14 @@ async def create_new_orders(self, symbol, _, state, **kwargs):
             else:
                 initial_available_funds = current_market_holding \
                     if side is trading_enums.TradeOrderSide.BUY else current_symbol_holding
-            # cancel existing DCA orders from previous iterations
-            existing_orders = [
-                order
-                for order in self.exchange_manager.exchange_personal_data.orders_manager.get_open_orders(symbol=symbol)
-                if not (order.is_cancelled() or order.is_closed()) and side is order.side
-            ]
+            
+            if self.trading_mode.cancel_open_orders_at_each_entry:
```

Comment:
we should init `existing_orders` at [] or in a ternary operator

Reply:
Right! Done